packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -614,6 +620,15 @@ export interface AddAutoScalingGroupCapacityOptions {
    * @default Duration.minutes(5)
    */
   readonly taskDrainTime?: Duration;
+
+  /**
+   * Automated Draining for Spot Instances running Amazon ECS Services
+   *
+   * @see https://docs.aws.amazon.com/AmazonECS/latest/developerguide/container-instance-spot.html
```

Comment:
Nit: for the format consistency the comment would better to be

Specify whether to enable Automated Draining for Spot Instances running Amazon ECS Services. For more information, see \[Using Spot Instances\](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/container-instance-spot.html).

Reply:
fixed https://github.com/aws/aws-cdk/pull/4360/commits/45bef1da7735bc55ff50668e70c7c3c9349b6944